README.md

```diff
@@ -71,6 +71,18 @@ Especially relevant for developers, there exists a basic test framework written
 python -m pytest -sv tests
 ```
 
+## Run examples
+
+A number of examples exist which document how the tools can be used. For a standalone
+usage convenient jupyter notebooks are available for each tool. To use them jupyter
+and related tools have to be installed in the development environment as follows:
+
+```shell
+python -m pip install jupyter
+python -m pip install jupyterlab
+python -m pip install jupyterlab_h5web
```

Comment:
Isn't jupyterlab_h5web enough alone, so the others are installed automatically as dependencies?

Reply:
When I worked on this branch in a local environment I was for some reason unable to get jupyterlab running when installing inside a fresh env using only jupyterlab_h5web, I made the same exercise right now and indeed jupyterlab_h5web again try to start jupyter lab out of the box without the two above does not work despite using the right local python3.10 env and not my conda base py3.9 env. Thus, I will keep the above suggestions to users for now as it reads above.

Reply:
thanks

Reply:
ran into this again today and learned that indeed for py310 the second line is not needed as the pip install jupyter will install jupyterlab and two other dependencies which the jupyterlab_h5web does not pick up